nix/file_handling.md

```diff
@@ -0,0 +1,192 @@
+# Creating, editing, removing files
+
+## Use a text editor
+
+There are a *lot* of different text editors and a *lot* of strong
+feelings about which of them is the best.
+
+You can use any editor you like, but you _must_ know how to use at least one
+terminal-friendly editor. In this workshop we are going to use `nano`. It's
+simple and easy to use.
+
+Make sure you're in your home directory (use `cd` and `pwd` to confirm) then type
+
+```console
+$ nano
+```
+
+This is a no-frills editor. Type something! How about a TODO list?
+
+```console
+TODO
+* [x] Learn how to navigate using the terminal
+* [ ] Learn how to create files
+* [ ] Learn about pipes and redirects
+```
+
+At the bottom you'll notice a bunch of different options but we are concerned
+with only two of them: `Write Out` (save) and `Exit`.
+
+The caret (`^`) means the Control key. To save the TODO list, hit `Ctrl+o`, type
+in a name (how about "TODO") and then hit `Enter`. `nano` will report that it
+wrote some number of lines.
+
+Now exit `nano` by typing `Ctrl+x`.
+
+Use `ls` to see what happened:
+
+```console
+$ ls -F
```

Comment:
Yes -- it's explained [here](https://github.com/barbagroup/essential_skills_RRC/pull/8/files#diff-92888dbbf81f3834d7e4c303bc8aec1fR51) which will come before this lesson (they're all out of order right now...)